rocAL/include/pipeline/commons.h

```diff
@@ -154,3 +154,15 @@ struct Timing {
     long long unsigned video_decode_time= 0;
     long long unsigned video_process_time= 0;
 };
+
+/*! \brief Tensor Last Batch Policies
+ These policies the last batch policies determine the behavior when there are not enough samples in the epoch to fill the last batch
+        FILL - The last batch is filled by either repeating the last sample or by wrapping up the data set.
+        DROP - The last batch is dropped if when there are not enough samples from the current epoch.
+        PARTIAL - The last batch is partially filled with the remaining data from the current epoch, keeping the rest of the samples empty. (currently this policy works similar to FILL in rocAL, PARTIAL policy needs to handled from python end)
```

Comment:
The rest of the samples are not empty right?

Reply:
The last batch is dropped if it cannot be fully filled with data from the current epoch

Reply:
Wordings taken from DALI description as is